app-misc/openrgb-plugin-effects/openrgb-plugin-effects-0.9-r1.ebuild

```
@@ -23,6 +23,7 @@ KEYWORDS="amd64"
 
 RDEPEND="
 	>=app-misc/openrgb-0.9:=[-qt6(-)]
+	<app-misc/openrgb-1.0
```

This would allow the dependency to be fulfilled by Qt6-based `app-misc/openrgb-0.9_p20250802` - which is not what you want, I guess?

It looks like this USEdep could also be cleaned up, now that the transition will not involve an IUSE qt6 (introduced in abcd889ece12a41ed5f8f729900a3125c80c1555 for transition) :)

```suggestion
	>=app-misc/openrgb-0.9:=
	<app-misc/openrgb-0.9_p20250802
```